Monika After Story/game/script-topics.rpy

```diff
@@ -10501,4 +10501,158 @@ label monika_hemispheres_nogets_snow:
     m 3hksdlb "At least you don't have to worry about shoveling it."
     call monika_snow_nogets_snow
     return
+    
+    default persistent._mas_pm_wants_children = None
+    # do you want children?
+    
+    default persistent._mas_pm_wants_1_child = None
+    # do you want one child?
+    
+    default persistent._mas_pm_wants_boy = None
+    # do you want a boy?
+    
+    default persistent._mas_pm_wants_girl = None
+    # do you want a girl?
+    
+    default persistent._mas_pm_wants_2_to_3_children = None
+    # do you want two to three children?
+    
+    default persistent._mas_pm_wants_many_children = None
+    # do you want many children?
+      
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="Children",category=['romance','monika','life'],prompt="Having Children",random=True))
+
+label monika_children:
+    if mas_isMoniLOVE():
```

Comment:
```suggestion
    if mas_isMoniLove():
```

Reply:
Also, everything that's supposed to be in this `if` statement needs to move up an indent level.